kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -56,8 +55,11 @@ def pad(x, pad_left, pad_right):
     """
     if (pad_left >= x.shape[-1]) or (pad_right >= x.shape[-1]):
         raise ValueError('Indefinite padding size (larger than tensor).')
+    
+    paddings = ( ((0, 0), ) * len(x.shape[:-1]))
```

Comment:
NICE

Reply:
Remove outermost parentheses.

Reply:
I believe we need the outer most paren so its in a tuple. Let me try it out however.